src/main/java/io/specto/hoverfly/junit/dsl/matchers/PlainTextMatcher.java

```diff
@@ -0,0 +1,6 @@
+package io.specto.hoverfly.junit.dsl.matchers;
+
+public interface PlainTextMatcher extends RequestMatcher {
```

Comment:
This is not exactly marker interface pattern, it extends from the root interface RequestMatcher, and just happens to be empty for now. Inheritance is something I try to avoid as well, interface would be better than some empty abstract classes to extend from.